routes/index.js

```diff
@@ -815,6 +815,8 @@ exports.api.datasources = (req, res) => {
 		const { uuid } = req.session || {}
 		const { tag } = req.body || {}
 
+		// FIXME if the entry already exists we will return 500 here (because of 'one')
```

Comment:
I noticed that while fixing an issue on my side. Not sure if the current behavior here is desired. @myjyby let me know if not -- I can quickly fix it in this pr, too.